[02-Oct-2024 15:16:05 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 6
[02-Oct-2024 15:17:52 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:17:52 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:17:53 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:17:53 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:17:53 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:17:53 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:17:53 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:17:53 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:18:02 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:18:02 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:18:30 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:20:37 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:20:37 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 9
[02-Oct-2024 15:23:47 America/Bogota] PHP Warning:  Undefined array key 1 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 7
[02-Oct-2024 15:27:36 America/Bogota] PHP Fatal error:  Uncaught Error: Call to protected method App\Controllers\BaseController::getParam() from global scope in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php:7
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 7
[02-Oct-2024 15:28:24 America/Bogota] PHP Warning:  Undefined variable $bc in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 7
[02-Oct-2024 15:28:24 America/Bogota] PHP Fatal error:  Uncaught Error: Call to a member function getParam() on null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php:7
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 7
[02-Oct-2024 15:28:25 America/Bogota] PHP Warning:  Undefined variable $bc in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 7
[02-Oct-2024 15:28:25 America/Bogota] PHP Fatal error:  Uncaught Error: Call to a member function getParam() on null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php:7
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 7
[02-Oct-2024 15:30:56 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:30:56 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:30:56 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:30:56 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:30:58 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:30:58 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:30:58 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:30:58 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 8
[02-Oct-2024 15:32:47 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:33:45 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:33:49 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:33:50 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:33:53 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:33:55 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:34:15 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:34:16 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:34:17 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:35:37 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 15:35:38 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 16:09:11 America/Bogota] Contrasena incorrecta
[02-Oct-2024 16:56:50 America/Bogota] PHP Fatal error:  Uncaught ValueError: explode(): Argument #1 ($separator) cannot be empty in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php:10
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php(10): explode('', 'eyJ0eXAiOiJKV1Q...')
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require('C:\\xampp\\htdocs...')
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 10
[02-Oct-2024 16:56:51 America/Bogota] PHP Fatal error:  Uncaught ValueError: explode(): Argument #1 ($separator) cannot be empty in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php:10
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php(10): explode('', 'eyJ0eXAiOiJKV1Q...')
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require('C:\\xampp\\htdocs...')
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 10
[02-Oct-2024 16:59:04 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[02-Oct-2024 16:59:06 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[12-Nov-2024 10:09:14 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[12-Nov-2024 10:09:51 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[12-Nov-2024 10:12:36 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 10
[12-Nov-2024 10:12:48 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 10
[12-Nov-2024 10:20:32 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:20:32 America/Bogota] PHP Fatal error:  Uncaught TypeError: App\Controllers\UserController::getLogin(): Argument #1 ($endPoint) must be of type string, null given, called in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22 and defined in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\UserController.php:13
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php(22): App\Controllers\UserController->getLogin(NULL)
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require('C:\\xampp\\htdocs...')
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\UserController.php on line 13
[12-Nov-2024 10:21:03 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:03 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:03 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:03 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:06 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:06 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:06 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:06 America/Bogota] PHP Warning:  Trying to access array offset on value of type null in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[12-Nov-2024 10:21:31 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[13-Nov-2024 11:12:48 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[13-Nov-2024 11:12:50 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 22
[13-Nov-2024 11:13:37 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:13:38 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:13:39 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:14:33 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:15:04 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 25
[13-Nov-2024 11:15:12 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 25
[13-Nov-2024 11:15:13 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 25
[13-Nov-2024 11:16:44 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:16:46 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:17:03 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:17:04 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 24
[13-Nov-2024 11:32:17 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 23
[13-Nov-2024 11:32:19 America/Bogota] PHP Warning:  Undefined variable $params in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 23
[13-Nov-2024 11:44:22 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[13-Nov-2024 11:44:23 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 11
[13-Nov-2024 11:52:01 America/Bogota] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function App\Controllers\UserController::getLogin(), 1 passed in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 21 and exactly 2 expected in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\UserController.php:13
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php(21): App\Controllers\UserController->getLogin('getLogin')
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require('C:\\xampp\\htdocs...')
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\UserController.php on line 13
[13-Nov-2024 11:52:12 America/Bogota] PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function App\Controllers\UserController::getLogin(), 1 passed in C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php on line 21 and exactly 2 expected in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\UserController.php:13
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\auth.php(21): App\Controllers\UserController->getLogin('getLogin')
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(21): require('C:\\xampp\\htdocs...')
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\UserController.php on line 13
[13-Nov-2024 12:24:58 America/Bogota] PHP Warning:  Array to string conversion in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 25
[13-Nov-2024 12:26:08 America/Bogota] PHP Fatal error:  Uncaught Error: Call to a member function addRoute() on string in C:\xampp\htdocs\curso_prueba\backend\public\index.php:23
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 23
[13-Nov-2024 12:26:10 America/Bogota] PHP Fatal error:  Uncaught Error: Call to a member function addRoute() on string in C:\xampp\htdocs\curso_prueba\backend\public\index.php:23
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 23
[13-Nov-2024 15:28:24 America/Bogota] PHP Fatal error:  Uncaught Error: Undefined constant "App\Routes\URL" in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:15
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(10): App\Routes\Router->matchRoute()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 15
[13-Nov-2024 15:29:25 America/Bogota] PHP Fatal error:  Uncaught Error: Undefined constant "App\Routes\URL" in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:15
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(10): App\Routes\Router->matchRoute()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 15
[13-Nov-2024 15:31:12 America/Bogota] PHP Fatal error:  Uncaught Error: Undefined constant "App\Routes\URL" in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:16
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(11): App\Routes\Router->matchRoute()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 16
[13-Nov-2024 15:34:17 America/Bogota] PHP Warning:  require_once(C:\xampp\htdocs\curso_prueba\backend\src\Routes/../public/config.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 4
[13-Nov-2024 15:34:17 America/Bogota] PHP Fatal error:  Uncaught Error: Failed opening required 'C:\xampp\htdocs\curso_prueba\backend\src\Routes/../public/config.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:4
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(576): include()
#1 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): Composer\Autoload\ClassLoader->loadClass('App\\Routes\\Rout...')
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 4
[13-Nov-2024 15:44:39 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method AuthController::method() in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:31
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 31
[13-Nov-2024 15:45:08 America/Bogota] PHP Fatal error:  Uncaught Error: Object of class AuthController could not be converted to string in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:30
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 30
[13-Nov-2024 15:45:09 America/Bogota] PHP Fatal error:  Uncaught Error: Object of class AuthController could not be converted to string in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:30
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 30
[13-Nov-2024 15:47:17 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method AuthController::method() in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:31
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 31
[13-Nov-2024 15:47:19 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method AuthController::method() in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:31
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 31
[13-Nov-2024 15:51:13 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 19
[13-Nov-2024 15:51:13 America/Bogota] PHP Fatal error:  Uncaught Error: Method name must be a string in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:31
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 31
[13-Nov-2024 15:51:16 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 19
[13-Nov-2024 15:51:16 America/Bogota] PHP Warning:  require_once(C:\xampp\htdocs\curso_prueba\backend\src\Routes/../Controllers/Controller.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 23
[13-Nov-2024 15:51:16 America/Bogota] PHP Fatal error:  Uncaught Error: Failed opening required 'C:\xampp\htdocs\curso_prueba\backend\src\Routes/../Controllers/Controller.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:23
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(11): App\Routes\Router->matchRoute()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 23
[13-Nov-2024 16:00:14 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 31
[13-Nov-2024 16:00:19 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 31
[13-Nov-2024 16:00:35 America/Bogota] PHP Warning:  Undefined array key 2 in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 31
[13-Nov-2024 16:11:52 America/Bogota] PHP Warning:  Undefined variable $data in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 55
[13-Nov-2024 16:15:12 America/Bogota] PHP Fatal error:  Uncaught Error: Unknown named parameter $Username in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:56
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(56): call_user_func_array(Array, Array)
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 56
[13-Nov-2024 16:15:13 America/Bogota] PHP Fatal error:  Uncaught Error: Unknown named parameter $Username in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:56
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(56): call_user_func_array(Array, Array)
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 56
[13-Nov-2024 16:15:33 America/Bogota] PHP Fatal error:  Uncaught Error: Unknown named parameter $Username in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:56
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(56): call_user_func_array(Array, Array)
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 56
[13-Nov-2024 16:32:19 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:32:57 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:33:11 America/Bogota] PHP Fatal error:  Uncaught Error: Class "Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:39:53 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:39:54 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:41:05 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:41:07 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:41:34 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:41:37 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:41:38 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:42:31 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:44:10 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:9
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): AuthController->__construct()
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 9
[13-Nov-2024 16:44:45 America/Bogota] PHP Fatal error:  Uncaught Error: Call to a member function connect() on null in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php:13
Stack trace:
#0 [internal function]: AuthController->getLogin(Array)
#1 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(56): call_user_func(Array, Array)
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\authController.php on line 13
[13-Nov-2024 16:50:02 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:52:42 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:52:43 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:52:43 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:52:44 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:54:31 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:54:34 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:54:35 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:56:54 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:56:55 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:59:46 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:59:47 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:59:48 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:59:58 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\Routes\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 16:59:59 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\Routes\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 17:02:46 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\DB\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:17
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 17
[13-Nov-2024 17:07:33 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\database\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:18
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 18
[13-Nov-2024 17:07:36 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\database\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:18
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 18
[13-Nov-2024 17:11:35 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\database\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:18
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 18
[13-Nov-2024 17:11:58 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\database\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:18
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 18
[13-Nov-2024 17:13:34 America/Bogota] PHP Warning:  Undefined property: App\Routes\Router::$data in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 57
[13-Nov-2024 17:13:34 America/Bogota] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #2 ($args) must be of type array, null given in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:57
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(57): call_user_func_array(Array, NULL)
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 57
[13-Nov-2024 17:14:31 America/Bogota] PHP Fatal error:  Uncaught Error: Unknown named parameter $Username in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:58
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(58): call_user_func_array(Array, Array)
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 58
[13-Nov-2024 17:14:32 America/Bogota] PHP Fatal error:  Uncaught Error: Unknown named parameter $Username in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:58
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(58): call_user_func_array(Array, Array)
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 58
[13-Nov-2024 17:18:07 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\database\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:16
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 16
[13-Nov-2024 17:18:39 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\database\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:16
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 16
[13-Nov-2024 17:18:40 America/Bogota] PHP Fatal error:  Uncaught Error: Class "App\database\Database" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:16
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 16
[14-Nov-2024 09:03:57 America/Bogota] PHP Warning:  require_once(vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 4
[14-Nov-2024 09:03:57 America/Bogota] PHP Fatal error:  Uncaught Error: Failed opening required 'vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php:4
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(576): include()
#1 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(8): Composer\Autoload\ClassLoader->loadClass('App\\DB\\Database')
#3 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(576): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): Composer\Autoload\ClassLoader->loadClass('App\\Routes\\Rout...')
#6 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 4
[14-Nov-2024 09:04:52 America/Bogota] PHP Warning:  require_once(./vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 5
[14-Nov-2024 09:04:52 America/Bogota] PHP Fatal error:  Uncaught Error: Failed opening required './vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php:5
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(576): include()
#1 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(8): Composer\Autoload\ClassLoader->loadClass('App\\DB\\Database')
#3 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(576): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): Composer\Autoload\ClassLoader->loadClass('App\\Routes\\Rout...')
#6 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 5
[14-Nov-2024 09:05:29 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method App\Routes\Router::run() in C:\xampp\htdocs\curso_prueba\backend\public\index.php:12
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 12
[14-Nov-2024 09:07:03 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method App\Routes\Router::run() in C:\xampp\htdocs\curso_prueba\backend\public\index.php:12
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 12
[14-Nov-2024 09:07:04 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method App\Routes\Router::run() in C:\xampp\htdocs\curso_prueba\backend\public\index.php:12
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 12
[14-Nov-2024 09:07:06 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method App\Routes\Router::run() in C:\xampp\htdocs\curso_prueba\backend\public\index.php:12
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 12
[14-Nov-2024 09:07:27 America/Bogota] PHP Fatal error:  Uncaught Error: Call to undefined method App\Routes\Router::run() in C:\xampp\htdocs\curso_prueba\backend\public\index.php:12
Stack trace:
#0 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\public\index.php on line 12
[14-Nov-2024 10:05:40 America/Bogota] PHP Fatal error:  Uncaught ArgumentCountError: print_r() expects at most 2 arguments, 3 given in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php:29
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(29): print_r(false, false, false)
#1 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(18): App\DB\Database->__construct()
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 29
[14-Nov-2024 10:06:27 America/Bogota] PHP Fatal error:  Uncaught ArgumentCountError: print_r() expects at most 2 arguments, 3 given in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php:30
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(30): print_r(false, false, false)
#1 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(19): App\DB\Database->connect()
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 30
[14-Nov-2024 10:06:28 America/Bogota] PHP Fatal error:  Uncaught ArgumentCountError: print_r() expects at most 2 arguments, 3 given in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php:30
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(30): print_r(false, false, false)
#1 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(19): App\DB\Database->connect()
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 30
[14-Nov-2024 10:06:29 America/Bogota] PHP Fatal error:  Uncaught ArgumentCountError: print_r() expects at most 2 arguments, 3 given in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php:30
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(30): print_r(false, false, false)
#1 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(19): App\DB\Database->connect()
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 30
[14-Nov-2024 10:26:36 America/Bogota] PHP Warning:  require_once(../../vendor/autoload.php): Failed to open stream: No such file or directory in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 4
[14-Nov-2024 10:26:36 America/Bogota] PHP Fatal error:  Uncaught Error: Failed opening required '../../vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php:4
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(576): include()
#1 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(10): Composer\Autoload\ClassLoader->loadClass('App\\DB\\Database')
#3 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(576): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\curso_prueba\backend\vendor\composer\ClassLoader.php(427): Composer\Autoload\{closure}('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): Composer\Autoload\ClassLoader->loadClass('App\\Routes\\Rout...')
#6 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 4
[14-Nov-2024 10:27:22 America/Bogota] PHP Fatal error:  Uncaught Dotenv\Exception\InvalidPathException: Unable to read any of the environment file(s) at [C:\xampp\htdocs\curso_prueba\backend\src\DB\.env]. in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php:68
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Dotenv.php(222): Dotenv\Store\FileStore->read()
#1 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(19): Dotenv\Dotenv->load()
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(18): App\DB\Database->__construct()
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php on line 68
[14-Nov-2024 10:27:49 America/Bogota] PHP Fatal error:  Uncaught Dotenv\Exception\InvalidPathException: Unable to read any of the environment file(s) at [C:\xampp\htdocs\curso_prueba\backend\src\DB\.env]. in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php:68
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Dotenv.php(222): Dotenv\Store\FileStore->read()
#1 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(20): Dotenv\Dotenv->load()
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(18): App\DB\Database->__construct()
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php on line 68
[14-Nov-2024 10:31:09 America/Bogota] PHP Fatal error:  Uncaught Dotenv\Exception\InvalidPathException: Unable to read any of the environment file(s) at [C:\xampp\htdocs\curso_prueba\backend\src\DB/../../\.env]. in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php:68
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Dotenv.php(222): Dotenv\Store\FileStore->read()
#1 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(20): Dotenv\Dotenv->load()
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(18): App\DB\Database->__construct()
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php on line 68
[14-Nov-2024 10:31:10 America/Bogota] PHP Fatal error:  Uncaught Dotenv\Exception\InvalidPathException: Unable to read any of the environment file(s) at [C:\xampp\htdocs\curso_prueba\backend\src\DB/../../\.env]. in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php:68
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Dotenv.php(222): Dotenv\Store\FileStore->read()
#1 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(20): Dotenv\Dotenv->load()
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(18): App\DB\Database->__construct()
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php on line 68
[14-Nov-2024 10:31:11 America/Bogota] PHP Fatal error:  Uncaught Dotenv\Exception\InvalidPathException: Unable to read any of the environment file(s) at [C:\xampp\htdocs\curso_prueba\backend\src\DB/../../\.env]. in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php:68
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Dotenv.php(222): Dotenv\Store\FileStore->read()
#1 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(20): Dotenv\Dotenv->load()
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(18): App\DB\Database->__construct()
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php on line 68
[14-Nov-2024 10:31:25 America/Bogota] PHP Fatal error:  Uncaught Dotenv\Exception\InvalidPathException: Unable to read any of the environment file(s) at [C:\xampp\htdocs\curso_prueba\backend\src\DB/../../\.env]. in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php:68
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Dotenv.php(222): Dotenv\Store\FileStore->read()
#1 C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php(20): Dotenv\Dotenv->load()
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(18): App\DB\Database->__construct()
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\vendor\vlucas\phpdotenv\src\Store\FileStore.php on line 68
[14-Nov-2024 10:37:17 America/Bogota] PHP Warning:  Undefined array key "APP_NAME" in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 26
[14-Nov-2024 10:37:18 America/Bogota] PHP Warning:  Undefined array key "APP_NAME" in C:\xampp\htdocs\curso_prueba\backend\src\DB\Database.php on line 26
[14-Nov-2024 11:26:27 America/Bogota] PHP Fatal error:  Uncaught Error: Class "AuthController" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:59
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 59
[14-Nov-2024 11:26:28 America/Bogota] PHP Fatal error:  Uncaught Error: Class "AuthController" not found in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:59
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#1 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 59
[14-Nov-2024 11:33:08 America/Bogota] PHP Fatal error:  Uncaught TypeError: method_exists(): Argument #1 ($object_or_class) must be of type object|string, bool given in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:54
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): method_exists(false, 'getLogin')
#1 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(27): App\Routes\Router->matchRoute()
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 54
[14-Nov-2024 11:33:09 America/Bogota] PHP Fatal error:  Uncaught TypeError: method_exists(): Argument #1 ($object_or_class) must be of type object|string, bool given in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php:54
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(54): method_exists(false, 'getLogin')
#1 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(27): App\Routes\Router->matchRoute()
#2 C:\xampp\htdocs\curso_prueba\backend\public\index.php(11): App\Routes\Router->__construct()
#3 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php on line 54
[14-Nov-2024 12:42:51 America/Bogota] PHP Warning:  foreach() argument must be of type array|object, null given in C:\xampp\htdocs\curso_prueba\backend\src\Config\ResponseHttp.php on line 126
[14-Nov-2024 12:42:57 America/Bogota] PHP Warning:  foreach() argument must be of type array|object, null given in C:\xampp\htdocs\curso_prueba\backend\src\Config\ResponseHttp.php on line 126
[14-Nov-2024 12:43:02 America/Bogota] PHP Fatal error:  Uncaught Exception: El parámetro 'category_id' no puede estar vacío. in C:\xampp\htdocs\curso_prueba\backend\src\Config\ResponseHttp.php:128
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(19): App\Config\ResponseHttp::validateParams(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(75): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Config\ResponseHttp.php on line 128
[14-Nov-2024 12:43:02 America/Bogota] PHP Fatal error:  Uncaught Exception: El parámetro 'category_id' no puede estar vacío. in C:\xampp\htdocs\curso_prueba\backend\src\Config\ResponseHttp.php:128
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(19): App\Config\ResponseHttp::validateParams(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(75): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Config\ResponseHttp.php on line 128
[14-Nov-2024 13:06:39 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.task' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:26
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(26): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 26
[14-Nov-2024 13:06:54 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[HY093]: Invalid parameter number: parameter was not defined in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:27
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(27): PDOStatement->execute(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 27
[14-Nov-2024 13:08:21 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`todolistapp`.`tasks`, CONSTRAINT `tasks_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `users` (`user_id`) ON DELETE CASCADE) in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:33
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(33): PDOStatement->execute(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 33
[14-Nov-2024 13:09:00 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1110 Column 'user_id' specified twice in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:30
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(30): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 30
[14-Nov-2024 13:10:21 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`todolistapp`.`tasks`, CONSTRAINT `tasks_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `users` (`user_id`) ON DELETE CASCADE) in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:33
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(33): PDOStatement->execute(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 33
[14-Nov-2024 13:11:30 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`todolistapp`.`tasks`, CONSTRAINT `tasks_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `users` (`user_id`) ON DELETE CASCADE) in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:33
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(33): PDOStatement->execute(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 33
[14-Nov-2024 13:11:33 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`todolistapp`.`tasks`, CONSTRAINT `tasks_ibfk_1` FOREIGN KEY (`user_id`) REFERENCES `users` (`user_id`) ON DELETE CASCADE) in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:33
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(33): PDOStatement->execute(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 33
[14-Nov-2024 13:14:45 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[HY093]: Invalid parameter number in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:46
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(46): PDOStatement->execute(Array)
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 46
[14-Nov-2024 13:17:16 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1110 Column 'user_id' specified twice in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:44
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(44): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 44
[14-Nov-2024 13:17:17 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1110 Column 'user_id' specified twice in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:44
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(44): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 44
[14-Nov-2024 14:43:52 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1110 Column 'user_id' specified twice in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:43:53 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42000]: Syntax error or access violation: 1110 Column 'user_id' specified twice in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:54:49 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:55:06 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:55:07 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:56:44 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:57:46 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:58:18 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 14:58:19 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 15:00:13 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 15:00:15 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 15:01:45 America/Bogota] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'todolistapp.tasks' doesn't exist in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:45
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(45): PDO->prepare('INSERT INTO tas...')
#1 [internal function]: App\Controllers\TaskController->createTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 45
[14-Nov-2024 16:13:51 America/Bogota] PHP Fatal error:  Uncaught Error: Class "TaskModel" not found in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php:16
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(73): App\Controllers\TaskController->__construct(Object(PDO))
#1 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#2 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php on line 16
[14-Nov-2024 17:09:08 America/Bogota] PHP Fatal error:  Uncaught Error: Undefined constant "App\Models\fieldsToUpdate" in C:\xampp\htdocs\curso_prueba\backend\src\Models\TaskModel.php:82
Stack trace:
#0 C:\xampp\htdocs\curso_prueba\backend\src\Controllers\taskController.php(27): App\Models\TaskModel->update(Array)
#1 [internal function]: App\Controllers\TaskController->updateTask(Array)
#2 C:\xampp\htdocs\curso_prueba\backend\src\Routes\Router.php(77): call_user_func(Array, Array)
#3 C:\xampp\htdocs\curso_prueba\backend\public\index.php(12): App\Routes\Router->run()
#4 {main}
  thrown in C:\xampp\htdocs\curso_prueba\backend\src\Models\TaskModel.php on line 82
